README.md

```diff
@@ -106,10 +135,56 @@ All commands are issued from the the root of the project folder.
  * (e.g. `<project_root>/platforms/ios/ThaliTest.xcodeproj`)
  * `~/Code/ThaliTest> open platforms/ios/ThaliTest.xcodeproj`
 
+### Unit Testing the Thali Cordova Plugin
+
+The Thali Cordova plugin uses the [Jasmine](http://jasmine.github.io/) Behavior Driven Development framework for JavaScript.
+
+#### Desktop Testing
+
+Testing is available on the desktop for the Thali Cordova Plugin which uses a mock object to simulate the native Cordova `Mobile` calls.
+
+To get started, install, Jasmine:
+```
+$ sudo [npm] install -g jasmine
+```
+
+Next, simply run Jasmine which then looks at the specifications in the `specs` folder.  
+```
+$ jasmine
+```
+
+#### Mobile Testing
+
+Testing is also available on the mobile devices as well.  This will use the regular `Mobile` calls directly through Cordova to talk to the underlying system.
+
+To get started, copy the files from `sample/android/www` or `sample/ios/www` to the `www` folder.
```

Comment:
This part confuses me. Why would we have a sample/android/www and sample/ios/www? I thought the whole point of having a single native UX layer in Node.js is that we have the exact same files for both iOS and android, at least at the Node.js layer. My expectation is that there would be a single directory called testing that contains all the test code and that the user would copy that code to the www folder.